docs/software/spack.md

```diff
@@ -0,0 +1,179 @@
+# Spack
+
+*Currently, Spack is only available on Alpine.* [Spack](https://spack.readthedocs.io/en/latest/) is a package manager designed for installing multiple configurations. Spack is designed for Supercomputing clusters, making it easy to install and configure software that may not currently be available in the Alpine software stack. The following documentation demonstrates how to activate & use the CURC Spack module and utilize your own Spack environments to install packages and compilers.
+
+## Using Spack on CURC
+You can follow these steps from a Research Computing terminal session via a compile job (`acompile`) or within a batch or interactive job.
+
+### Activate and Use the CURC Spack Module
+
+Run the following command to load the Spack module:  
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ module load spack/0.20.1
+```
+
+You can confirm that spack has been loaded and find a list of useful spack commands by running the `spack` command: 
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack
```

Comment:
```suggestion
[johndoe@c3cpu-c11-u17-2 ~]$ spack help --all
```

If you want to provide all of the options for Spack, I would go with my suggestion.